src/main/java/ledger/user_interface/ui_controllers/LoginPageController.java

```diff
@@ -105,21 +107,9 @@ private void openCreateFilePopup() {
         this.createModal(scene, "Create New File");
     }
 
-    /**
-     * Sets the internal file path variable.
-     *
-     * @param path Path to use
-     */
-    public void setFilePath(String path) {
-        this.filePath = path;
-    }
-
-    /**
-     * Sets the file button's text.
-     *
-     * @param name String to set the text to
-     */
-    public void setFileBtnText(String name) {
-        this.chooseFileBtn.setText(name);
+    public void setChosenFile(File file) {
+        this.filePath = file.getAbsolutePath();
+        this.chooseFileBtn.setText(file.getName());
+        this.password.requestFocus();
```

Comment:
I guess I don't really care, but you already have the data inside the fxml elements stored in the class. The variables are repetitive. 

Either way is fine by me though